src/System.ComponentModel.Composition.Registration/pkg/System.ComponentModel.Composition.Registration.pkgproj

```diff
@@ -2,12 +2,9 @@
     <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
     <ItemGroup>
     <ProjectReference Include="..\ref\System.ComponentModel.Composition.Registration.csproj">
-      <SupportedFramework>netcoreapp3.0;net45</SupportedFramework>
+      <SupportedFramework>netcoreapp3.0</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.ComponentModel.Composition.Registration.csproj" />
-    <InboxOnTargetFramework Include="net45">
```

Comment:
I think we needed to remove this as well in order to remove ```net45```. 
Does it look okay to you @ericstj ?

Reply:
Can you leave it in and instead declare support for net45 as follows:
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/package-projects.md#applicability-validation
```
<SupportedFramework Include="net45" Version="4.0.0.0" />
```

Reply:
FWIW this won't be needed once we can harvest from a previous release.  Then we'll assert the support based on the harvested package.